public/static/locales/en/data.json

```diff
@@ -107,11 +111,13 @@
     "pathListPatternMatchLbl": "Include only file / folder path(s) when file / folder name matches this <pattern>pattern</pattern>:",
     "pathListPatternMatchEmptyText": "e.g.: \\.csv$",
     "permissions": "Permissions",
+    "preview": "Preview",
     "publicLink": "Public Link",
     "publicLinks": "Public Link(s)",
     "publicLinkWarning": "Note: Anyone with access to a file's public link can access that file.",
     "publicLinkError": "Unable to get public link(s)",
     "read": "Read",
+    "readOnlyFileTooltip": "This file is read-only. It may be due to one of the following reasons: 1. This file is larger than {{maxEditSize}} bytes. 2. You don't have permission to edit this file. 3. Editing is not supported for this type of file.",
```

Comment:
Might be a little more readable for users if the max size was in MB instead of bytes?